crates/nu-protocol/src/engine/engine_state.rs

```diff
@@ -1619,6 +1619,26 @@ impl<'a> StateWorkingSet<'a> {
         next_id
     }
 
+    pub fn get_all(&self) -> Vec<String> {
```

Comment:
It would be nice if this `get_all` function were named more verbosely. My first question was, "get all what?".

Reply:
All defined variables by `let`

Reply:
Right, my point was the name of the function is not specific enough.